postgres-driver/postgres_driver.go

```diff
@@ -21,6 +21,8 @@ var (
 
 // PostgresDriver struct handler for PostgresDB related functions
 type PostgresDriver struct {
+	Notification chan *Notification
+	connString   string
```

Comment:
The connection string should not be leaked into this driver. Addressing the review comment on passing a notifier through an interface will also resolve this.